tinkerbell/stack/templates/hook.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.stack.hook.enabled }}
+{{- $hookDownloadJobEnabled := eq (include "stack.hookDownloadJobEnabled" .) "true" -}}
+{{- if and .Values.stack.hook.enabled $hookDownloadJobEnabled }}
```

Comment:
Its hard to tell where the corresponding `{{- end }}` is for this. mind putting the corresponding`{{- end }}` at the same level indent?

Reply:
All right